pages/portfolio.html

```diff
@@ -5,8 +5,53 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
-<body>
```

Comment:
Here you only deleted the _open_ tag for the body...  again, you should try to avoid that.  All of your content _should_ be placed in the body.  

The browser will guess at that and try to add it back in but might do it wrong.